UI/win-update/updater/updater.cpp

```diff
@@ -1292,10 +1295,17 @@ static bool Update(wchar_t *cmdLine)
 		if (argv) {
 			for (int i = 0; i < argc; i++) {
 				if (wcscmp(argv[i], L"Portable") == 0) {
+					// Legacy OBS
+					bIsPortable = true;
+					break;
+				} else if (wcsncmp(argv[i], L"--branch=", 9) ==
+					   0) {
+					branch = argv[i] + 9;
+				} else if (wcscmp(argv[i], L"--portable") ==
```

Comment:
`argv[i]` is a null terminated string, for `--branch=` this contains both the name of the parameter as well as its value, so we only want to compare the first 9 characters. But since `--portable` is a flag we can simply compare the whole string and do not need to specify a maximum length.

Reply:
Fair enough.